src/Compilers/Core/Portable/Emit/OperandType.cs

```diff
@@ -0,0 +1,37 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+namespace Microsoft.CodeAnalysis.Emit
+{
+
+    using System;
+#pragma warning disable RS0016 // Add public types and members to the declared API
+
+    [System.Runtime.InteropServices.ComVisible(true)]
+    public enum OperandType
```

Comment:
Removing dependency on System.Reflection.Emit contract sounds reasonable. Is this the only type from Ref.Emit we are using in the compilers?

We can temporarily add internal FILETYPE as well, but let's link the Xamarin bug that was filed.